packages/data/addon/adapters/facts/bard.ts

```diff
@@ -150,8 +151,15 @@ export default class BardFactsAdapter extends EmberObject implements NaviFactAda
       );
     }
 
-    const [start, end] = timeFilter.values;
+    let [start, end] = timeFilter.values as string[];
 
+    const endMoment = end ? moment.utc(end) : undefined;
+    if (endMoment?.isValid()) {
+      end = endMoment.add(1, getPeriodForGrain(filterGrain)).toISOString();
+    }
+
+    start = start.replace('T00:00:00.000Z', '');
```

Comment:
How would this work with Fili if you had `T00:00:00.000Z` in the value with an hourly grain?

Reply:
It wouldn't (I was planning to come back to fix `hourly` as a whole), but it looks like we can update this to just remove the `Z` because [timeZone has to be passed in as a param](https://github.com/yahoo/fili/blob/7c022c7368f7c99a83f4fe5b36731783af48a8b5/docs/end-user-api.md#time-zone)